doc/radosgw/multisite.rst

```diff
@@ -0,0 +1,710 @@
+to choose a format you prefer to set these names. So in summary:
+
+- Master Zonegroup: United States ``us``
+- Master zone: United States, East Region 1: ``us-east-1``
+- Secondary zone: United States, East Region 2: ``us-east-2``
+- Secondary zone: United States, West Region: ``us-west``
+
+This will be a part of a larger realm, say ``gold``. The zones ``us-east-1`` and
+``us-east-2`` are part of the same ceph cluster, of which we'll make
+``us-east-1`` as the primary. ``us-west`` will be in a different ceph cluster.
+
+Creating a realm
+----------------
+
+Configure a realm called ``gold``, and also make this the default ::
+
+  # radosgw-admin realm create --rgw-realm=gold --default
+  {
+    "id": "4a367026-bd8f-40ee-b486-8212482ddcd7",
+    "name": "gold",
+    "current_period": "09559832-67a4-4101-8b3f-10dfcd6b2707",
+    "epoch": 1
+  }
+
+
+Note that every realm has an id, which allows for flexibility like renaming the
+realm later, should the need arise. The ``current_period`` changes whenever we
+change anything in the master zone. The ``epoch`` is incremented when there's a
+change in configuration of the master zone, which results in a change of current
+period.
+
+Deleting the default zonegroup
+------------------------------
+
+A simple installation of radosgw would assume the default zonegroup called
+"default", since we no longer need this we begin by removing the default
+zonegroup::
+
+  # radosgw-admin zonegroup delete --rgw-zonegroup=default
+
+
+Creating a master zonegroup
+---------------------------
+
+We'll be creating a zonegroup called ``us`` as a master zonegroup. A master
+zonegroup will be in control of the zonegroup map and propagate changes to the
+rest of the system. We will also set this zonegroup as the default, which allows
+explicitly mentioning the ``rgw-zonegroup`` switch for later commands. We also
+make this the default zonegroup
+
+::
+
+   # radosgw-admin zonegroup create --rgw-zonegroup=us --endpoints=http://rgw1:80 --master --default
+   {
+    "id": "d4018b8d-8c0d-4072-8919-608726fa369e",
+    "name": "us",
+    "api_name": "us",
+    "is_master": "true",
+    "endpoints": [
+        "http:\/\/rgw1:80"
+    ],
+    "hostnames": [],
+    "hostnames_s3website": [],
+    "master_zone": "",
+    "zones": [],
+    "placement_targets": [],
+    "default_placement": "",
+    "realm_id": "4a367026-bd8f-40ee-b486-8212482ddcd7"
+    }
+
+Alternatively, we make this zonegroup as the default zonegroup via the following
+command ::
+
+  # radosgw-admin zonegroup default --rgw-zonegroup=us
+
+
+Creating a master zone
+----------------------
+
+Next we create a zone, and make it as the default zone. Note that for metadata
+operations like user creation you would want to use this zone. We also add it to
+the zonegroup
+
+::
+
+   # radosgw-admin zone create --rgw-zonegroup=us --rgw-zone=us-east-1 --endpoints=http://rgw1:80 --access-key=$ZONE_ACCESS_KEY --secret=$ZONE_SECRET_KEY --default --master
```

Comment:
You didn't define `$ZONE_ACCESS_KEY` earlier, should this be `$SYSTEM_ACCESS_KEY` instead?